### ISSUE TITLE
Manage stylesheet from .proton

### ISSUE THREAD

**Issue by mythmon** [dvcrn/proton#201]
As far as I can tell, there isn't a way to manage `styles.less` from `.proton`, unlike all the other configuration options for Atom. One of the main things I like about Proton is that all the configuration happens from a single file, which makes it dotfile friendly.

**Comment by dvcrn** [comment]
yes it is currently not possible to manage stylesheets. One idea we had was to generate collect all css files and do something like:

```javascript
var style = document.createElement('style');
style.type = 'text/css';
style.innerHTML = '.cssClass { color: #F00; }';
document.getElementsByTagName('head')[0].appendChild(style);
```

PRs are as always welcome 😄

**Comment by richarddewit** [comment]
Then the same thing should go for `snippets.cson` and `init.coffee`. But I would vote against it.

I just happen to have an Atom folder in my dotfiles repo containing `.proton` and I've simply added `styles.less`, `snippets.cson` and `init.coffee` to it. You already can edit them with <kbd>SPC f e s</kbd>, <kbd>SPC f e p</kbd> and <kbd>SPC f e i</kbd> respectively.  Let Atom handle those. In case you remove Proton, you won't end up with your custom styles removed along with it.
